### ISSUE TITLE
a dimension problem in baselines/dpd_ddpg/ddpg.py

### ISSUE THREAD

**Issue by zhanghongjie101** [datamllab/dual-policy-distillation#2]
I found a problem in baselines/dpd_ddpg/ddpg.py (line 196): self.actor_dis_loss = tf.reduce_sum(tf.math.multiply(self.weights, tf.reduce_mean(tf.square(self.actor_tf - self.dis_actions), axis=1)))
The shape of self.weights is (64, 1), and the shape of tf.reduce_mean(tf.square(self.actor_tf - self.dis_actions), axis=1)) is (64, ). so, the  multiply is not same as the core idea. I think you should use 'keep_dims=True' in tf.reduce_mean.
